librocksdb_sys/crocksdb/c.cc

```diff
@@ -3779,4 +3798,94 @@ int crocksdb_keyversions_type(const crocksdb_keyversions_t *kvs, int index) {
   return kvs->rep[index].type;
 }
 
+struct SstFileSeqNoModifier {
+  SstFileSeqNoModifier(Env *env, ColumnFamilyData *cfd):env_(env), cfd_(cfd) { }
+
+  Status ModifySeqNo(std::string file) {
```

Comment:
This function does too many things, better to extract functions to open the file, find the offset of the seqno, and update the seqno.